src/ui-components/QuickView.tsx

```
@@ -263,7 +268,7 @@ const QuickView = ({
 							</div>
 						</Tooltip>
 					)}
-					{loginUserId && !isNaN(loginUserId) && (
+					{!!loginUserId && !isNaN(loginUserId) && (
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)